devtools/cli/src/main/java/io/quarkus/cli/common/ListFormatOptions.java

```diff
@@ -18,13 +18,14 @@
     boolean origins = false;
 
     public String getFormatString() {
-        String formatString = "name";
```

Comment:
I dislike the default being present here because it makes it impossible to detect that no format has been set. (I wanted to be able to show the hint "Append `--full` to get more info" only if no format has been set by a user.)

The overall behavior didn't change, as I also modified the switch statement in the `ListExtensionsCommandHandler` to make the "name" format the default option there (which previously used to be "concise", but would always be overridden).

Having default here seems extraneous because:
* in Maven ListExtensionMojo it's overridden by its' own default "concise",
* in Gradle QuarkusListExtension it's overridden by its' own default "concise",
* for CLI the ListExtensionsCommandHandler has been modifed to use "name" as default if nothing is set.